app/_hub/kong-inc/ai-rate-limiting-advanced/how-to/_index.md

```diff
@@ -34,10 +34,10 @@ Protect your LLM service with rate limiting. It will analyze query costs and tok
 
 ```sh
 curl -i -X POST http://localhost:8001/services/example-service/plugins \
-  --data name=ai-rate-limiting-advanced \
-  --data config.limit=100,10000 \
-  --data config.window_size=60,3600 \
-  --data config.sync_rate=10
+  --data 'name=ai-rate-limiting-advanced' \
+  --data 'config.llm_providers[1].name=openai' \
+  --data 'config.llm_providers[1].limit=100' \
+  --data 'config.llm_providers[1].window_size=3600'
```

Comment:
```suggestion
  --data 'config.llm_providers[1].limit=100' \
  --data 'config.llm_providers[2].limit=10000' \
  --data 'config.llm_providers[1].window_size=3600'
  --data 'config.llm_providers[2].window_size=60'
```
I'm not 100% sure this is correct, but if we want to include both values from the original example, then we'd probably do this.

Feel free to ignore if that's unnecessary or doesn't work.